recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,25 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
-    movie_ratings = None
-    '''
-    complete the code to compute a variable movie_ratings
```

Comment:
The line is too long and should be split for better readability. Consider splitting it into multiple lines.